src/configuration/index.ts

```diff
@@ -33,6 +34,7 @@ export const defaultOptions: IRequiredConfigOptions = {
 	throwOnError: true,
 	allowRuntimeChangesInEnv: ['test'],
 	defaultNodeEnv: 'development',
+	loadFilesFromEnv: [NODE_ENV, 'SC_INSTANCE'],
```

Comment:
The file ending .json and path is added later? What is mean with NODE_ENV='NODE_ENV'?

Maybe loadFilesFromEnv is confused. This are only the fileNames, that should loaded later